Mathlib/Algebra/Order/Monoid/Prod.lean

```diff
@@ -41,4 +42,30 @@ instance [CanonicallyOrderedMonoid α] [CanonicallyOrderedMonoid β] :
     (inferInstance : ExistsMulOfLE _) with
       le_self_mul := fun _ _ ↦ ⟨le_self_mul, le_self_mul⟩ }
 
+namespace Lex
+
+@[to_additive]
+instance orderedCommMonoid [OrderedCommMonoid α]
+    [CovariantClass α α (· * ·) (· < ·)] [OrderedCommMonoid β] :
+    OrderedCommMonoid (α ×ₗ β) where
+  mul_le_mul_left := fun x y hxy z => ((le_iff _ _).1 hxy).elim
```

Comment:
Something like
```suggestion
  mul_le_mul_left x y hxy z := ((le_iff _ _).1 hxy).elim
```
should work, here and elsewhere. If the arguments are implicit, you'll need `{x y} hxy z` or similar instead.

Reply:
Thanks, done! Wondering whether it can be made to work with `{ }`, not just `where`.